src/core/parser/grammar.json

```diff
@@ -0,0 +1,150 @@
+{
+  "operators": [
+    [ "left", "Comma" ],
+    [ "left", "EqualToOrAssign", "AddAssign", "MinusAssign", "MultiplyAssign", "DivideAssign" ],
+    [ "left", "Not" ],
+    [ "left", "Or" ],
+    [ "left", "And" ],
+    [ "left", "Xor" ],
+    [ "left", "EqualTo", "GreaterThan", "GreaterThanOrEqualTo", "LessThan", "LessThanOrEqualTo", "NotEqualTo" ],
+    [ "left", "Add", "Minus" ],
+    [ "left", "Multiply", "Exponent", "Divide" ],
+    [ "left", "UnaryMinus" ],
+    [ "left", "LeftParen", "RightParen" ]
+  ],
+  "start": [ "node" ],
+  "bnf": {
+    "node": [
+      [ "statements EndOfInput", "return $1.flat();" ]
+    ],
+    "statements": [
+      [ "conditionalBlock", "$$ = [$1];" ],
+      [ "statements conditionalBlock", "$$ = $1.concat($2);" ],
+      [ "statement", "$$ = [$1];" ],
+      [ "statements statement", "$$ = $1.concat([$2]);" ]
+    ],
+    "statement": [
+      [ "text", "$$ = $1" ],
+      [ "shortcut", "$$ = [$1];" ],
+      [ "genericCommand", "$$ = [$1];" ],
+      [ "assignmentCommand", "$$ = [$1];" ],
+      [ "jumpCommand", "$$ = [$1];" ],
+      [ "stopCommand", "$$ = [$1];" ],
+      [ "statement Comment", "$$ = $1;" ],
+      [ "statement hashtags", "$$ = $1.map(s => Object.assign(s, { hashtags: $2 }));" ],
```

Comment:
Many changes in this grammar. This change is admittedly janky as heck. However, since hashtags can come after any kind of statement, this is semantically appropriate. It's just a really weird time to mutate an object. I think it's harmless enough though.